TIP-0003/README.md

```diff
@@ -0,0 +1,273 @@
+  
+  - if $g$ is a group identifier and $S$ is the set of group fungible series, then the sum of all tokens with identifiers ($g$, $s$), where $s\in S$, in the input is equal to the number of tokens with identifier ($g$, $s$), where $s\in S$.
+  
+- *Check Assets Quantity Descriptors*: Quantity descriptors regulate how UTXOs of an asset are split or merged. This is different from alloys, as the assets being merged or split always share the same group and series identifier. Let ($g$, $s$) be an asset identifier, all the following statements must be true:
+  - If  ($g$, $s$)'s quantity descriptor is `immutable`, then, for each input UTXO containing  ($g$, $s$) with quantity $n$ , there must be exactly one output containing  ($g$, $s$) with quantity $n$.
+  - If  ($g$, $s$)'s quantity descriptor is `accumulator`, then, for each input UTXO containing  ($g$, $s$) with quantity $n$ , it exists one output containing  ($g$, $s$) with quantity  $m \geq n$ .
+  - If  ($g$, $s$)'s quantity descriptor is `fractionable`, then, for each input UTXO containing  ($g$, $s$) with quantity $n$ ,  one or more outputs containing  ($g$, $s$) whose sum is equal to $n$.
+
+#### Merging Assets
+
+Merging assets mean that we take to assets that are fungible at one level but not the other and merge them into the same UTXO.  This requires the submission of a merging transaction to the node. To support this kind of transactions, the following validations need to be performed on the transaction.
+
+- *Check merging assets:* Let $A$ be a Asset Merging Statement to merge UTXOs $u_0$, $u_1$, ..., $u_{n-1}$ into index $i$. All the following statements are true:
+  - *All UTXOs share the same fungibility type:* There is a fungibility type $ft$, s.t. for all $u_i$, the fungibility type of $u_i$ is $ft$.
+  - *All UTXOs share the same quantity descriptor:*  There is a quantity descriptor $qd$, s.t. for all $u_i$, the quantity descriptor of $u_i$ is $qd$.
+  - The asset created at index $i$ has a seriesAlloy if the assets are group fungible, or a groupAlloy if the assets are series fungible. In addition to the actual groups or series, the Merkle tree also considers the quantity of each series or group in the alloy.
+  - The Merkle root in the merged asset corresponds to the values in the input UTXOs.
+
+#### Splitting Assets
+
+Splitting assets is the opposite of merging them.  This requires the submission of a splitting transaction to the node. To support this kind of transactions, the following validations need to be performed on the transaction.
+
+- *Check splitting assets:* Let $A$ be a Asset Splitting Statement to split UTXO $u$ into indexes $i_0$, .., $i_{n - 1}$ . All the following statements are true:
+  - All TXOs in the output have the same fungibility type as the one in the original UTXO $u$.

```

Comment:
I would avoid introducing a new term (TXOs) at this point, and just say `all transactions in the output`